soh/CMakeLists.txt

```diff
@@ -306,6 +323,37 @@ if (BUILD_REMOTE_CONTROL)
     else()
         set(SDL2-NET-INCLUDE ${SDL_NET_INCLUDE_DIRS})
     endif()
+
+    set(OPENSSL_USE_STATIC_LIBS ON)
+
+    if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+        file(STRINGS /etc/os-release distro REGEX "^NAME=")
+        string(REGEX REPLACE "NAME=\"(.*)\"" "\\1" distro "${distro}")
+
+        if(${distro} MATCHES "Fedora Linux")
+            set(OPENSSL_USE_STATIC_LIBS OFF)
+        endif()
+    endif()
```

Comment:
What's the motivation behind overriding this manually rather than the builder/packager setting it?

Reply:
thinking this one through, we have to account for dev machines and release builds on each platform

i think removing `set(OPENSSL_USE_STATIC_LIBS ON)` from here would work, just need to think through each part of this table

| platform | release/dev | how to handle it |
| - | - | - |
| windows | release | use `-DOPENSSL_USE_STATIC_LIBS` in the github actions workflow |
| windows | dev | not sure |
| linux | release | *maybe* use `-DOPENSSL_USE_STATIC_LIBS` in the workflow, not sure if having the flag on or off will be better for appimages |
| linux | dev | just not using static libs should be fine |
| mac | release | not sure |
| mac | dev | not sure |